test/basic.html

```diff
@@ -71,89 +62,58 @@
         emptyHeight = emptyDemo.getBoundingClientRect().height;
       });
 
-      test('can render native elements', function() {
-        var element = document.getElementById('nativeDemo');
-
-        // Render the distributed children.
-        Polymer.dom.flush();
-
-        var rect = element.getBoundingClientRect();
-
-        expect(rect.height).to.be.greaterThan(emptyHeight);
-
-        // The demo is rendered in the light dom, so it should exist, and
-        // it should respect the demo element's attributes, and not make up
-        // new ones.
-        var input = Polymer.dom(element).querySelector('input')
-        expect(input).to.be.ok;
-        expect(input.disabled).to.be.true;
-        expect(input.checked).to.be.false;
-
-        var markdownElement = element.$.marked;
-        expect(markdownElement.markdown).to.be.equal('```\n\n<input disabled>\n\n```');
-      });
-
-      test('can render custom elements', function() {
-        var element = document.getElementById('customDemo');
-
-        // Render the distributed children.
-        Polymer.dom.flush();
+      test('can render native elements', function(done) {
```

Comment:
All elements are declared w/o using test-fixture, so you shouldn't need to wait for distribution in each test, but rather wait 1ms before even starting the tests
Really a nit, but that would save like 2ms